<launch>

    <include file="$(find sensors)/launch/usv_simulation.launch" />

    <node name="cu_cr" pkg="sensors" type="cu_cr.py" >
        <param name = "desired_u" value = "0.7" />
        <param name = "desired_r" value = "8" />
    </node>

    <node name="obs_finite_time_asmc" pkg="sensors" type="obs_finite_time_asmc" >
        <param name = "t_c" value = "3" />
		<param name = "mu_u" value = "0.05" />
		<param name = "mu_r" value = "0.05" />
		<param name = "epsilon" value = "0.003" />
		<param name = "L1_u" value = "0.0" />
		<param name = "L1_dot_u" value = "0.0" />
		<param name = "L1_r" value = "0.0" />
		<param name = "L1_dot_r" value = "0.0" />
		<param name = "lambda_u" value = "0.01" />
		<param name = "lambda_r" value = "0.01" />
		<param name = "K2_u" value = "0.1" />
		<param name = "K2_r" value = "0.1" />
		<param name = "ro" value = "0.1" />
		<param name = "k_u" value = "2" />
		<param name = "k_r" value = "2" />
		<param name = "Kmin_u" value = "0.01" />
		<param name = "Kmin_r" value = "0.01" />
    </node>

</launch>
